main.tex

```diff
@@ -21,11 +21,13 @@
 \usepackage{minted}
 \usepackage{tikz}
 
+
 % Citation packages and options
 \usepackage{csquotes}
 \usepackage[
   backend=biber,
-  style=apa
+  style=numeric,
+  sorting=none
```

Comment:
Please remove this sorting parameter, the standard setting should be OK.

Reply:
We want the references be in the order of appearance...
Does it do that also without `sorting=none` ?

Reply:
You don't have a say, @danielrutz and @lschuermann are the editors ;)

Reply:
APA VI says that literature must be sorted alphabetically. I will not accept this change.